templates/blog/post_detail.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
+{% load crispy_forms_tags %}
+
+{% block content %}
+<h1>{{ post.title }}</h1>
+<p>{{ post.content }}</p>
+<h3>{{ post.commentary_count }} comment{{ post.commentary_count|pluralize }}:</h3>
+<ul>
+    {% for comment in post.commentary.all %}
```

Comment:
There is a potential issue here. The template is trying to access `post.commentary.all`, but it should be `post.comments.all` if the related name for comments in the Post model is `comments`. Please verify the related name in the model.